engine/src/universe_util.cpp

```diff
@@ -185,17 +185,20 @@ void saveGame(const string &savename) {
 }
 
 void showSplashScreen(const string &filename) {
```

Comment:
I spent a bit of time looking at this and `showSplashScreen` is called several times:
1. Twice in `main.cpp` with `filename=""`.
2. Once in `universe.cpp` with `filename=""`.
3. Once in `basecomputer.cpp` with `filename=""`.
4. Once in activateProgressScreen in showProgress.py with some data from VS.vsConfig.

The last one will almost surely break because it's reading from the old config with 

`return vs_config->getVariable(category, option, def);`

It is also called several times in seven places with a non-empty file.